tests/utils.py

```diff
@@ -72,3 +73,8 @@ def stub_open(*args, **kwargs):
 
     with patch("__builtin__.open", stub_open):
         yield mock_open, mock_file
+
+
+def get_unsupported_sys_versioninfo() -> tuple:
```

Comment:
what is the purpose of this method? I see it's being called in a test, but why? :) thanks

Reply:
This is a mock to return an unsupported version in the versioninfo() sys call.

Reply:
I'll add a docstring.

Reply:
@hellno added docstring.